core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.spec.ts

```diff
@@ -121,6 +124,18 @@ describe('SavedWorkflowSectionComponent', () => {
   //     .toEqual([2, 3, 4, 6, 8]);
   // });
 
+  // it('workflowDuplicateTest', () => {
+  //   component.workflows = [];
+  //   component.workflows = component.workflows.concat(TestCase);
+  //   component.onClickDuplicateWorkflow(workflows[3]);
+  //   const DuplicatedCaseContent = component.workflows.map(item => item.content);
+  //   const DuplicatedCaseName = component.workflows.map(item => item.name);
+  //   expect(DuplicatedCaseContent)
+  //     .toEqual(['{}', '{}', '{}', testContent , '{}', testContent]);
+  //   expect(DuplicatedCaseName)
+  //     .toEqual(['project 1', 'project 2', 'project 3', 'project 4', 'project 5', 'project 4_copy']);
+  // });
+
```

Comment:
why are they commented?

Reply:
> LGTM. What is happening to the test cases?

It is testing that after performing the onClickDuplicate function on a certain workflow , a copy of a workflow with the same content and name = previous name + '_copy' will appear in the frontend.

Reply:
> why are they commented?

It is commented just because the other testcases in the file are all commented, so I am just trying to follow the spec. I guess when we need all these test we can undo the comment? (I am actually also wondering why all the tests are commented here in this file though)